packages/itwin/measure-tools/vite.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig } from "vite";
+
+import react from "@vitejs/plugin-react";
+
+export default defineConfig({
+  plugins: [react()],
+  // look for all static in the dist folder
+  publicDir: "./public",
+  assetsInclude: "./public/*",
+  test: {
+    globals: true,
+    environment: "jsdom",
+    setupFiles: ["src/test/measure-tools/_Setup.test.ts"],
+    include: ["src/test/measure-tools/**/*.test.*"],
+    exclude: ["src/test/measure-tools/_Setup.test.ts"],
+    coverage: {
+      exclude: ["lib/*", "src/test/*"],
+      include: ["src/*"],
+      provider: "istanbul",
```

Comment:
I recommend using v8 instead of istanbul, see [this](https://github.com/iTwin/itwinjs-core/pull/7252) for explanation on why

Reply:
Done